examples/example_cluster_aws.go

```diff
@@ -37,7 +37,7 @@ func main() {
 
 	// -- 1. Get first project
 	projects, response, err := sdk.ProjectsApi.ListProjects(ctx).
-		IncludeCount(false).Execute()
+		IncludeCount(false).ItemsPerPage(1).Execute()
```

Comment:
verifying, was this change necessary? 1 should already be the default

Reply:
Default is 100. Page number is 1. Removing it as it doesn't seem to make much difference in example